.circleci/config.yml

```diff
@@ -34,77 +24,88 @@ jobs:
           command: |
             export RUNNING_IN_ARTMAN_DOCKER=True
             rm -rf gapic-generator/.git/
-            gapic-generator/gradlew -p gapic-generator fatJar createToolPaths install build -x test
+            gapic-generator/gradlew -p gapic-generator fatJar createToolPaths install build -x test -x javadoc
       - run:
-          name: Make reports dir
+          name: Make reports directory
           command: mkdir -p reports
+      - run:
+          name: Generate all client libraries
```

Comment:
I am leaning towards thinking it would be better for this to split into generating the client libraries per language. My rationale is that running the test for an API in python should not depend on whether the java generation failed. Thoughts?

Reply:
That makes sense. artman requires the artman docker image, so generating the clients has to happen in seperate jobs and containers from testing the clients. I think it would be a bit much to spin up 7 containers with artman to generate all the language clients, so one optimization could be to conditionally run the test-language-client job when the generation in that language passes.

I'll try that approach out, and hopefully that fits the model where "running the test for an API in python should not depend on whether the java generation failed".